### ISSUE TITLE
Error calloc invocation in src/image.c

### ISSUE THREAD

**Issue by kangmingxuan** [pjreddie/darknet#181]
Hi guys,

My friend turned to me today for a little piece of code in image.c, actually the function `load_alphabet`. 

```C
image **load_alphabet()
{
    int i, j;
    const int nsize = 8;
    image **alphabets = calloc(nsize, sizeof(image));
    for(j = 0; j < nsize; ++j){
        alphabets[j] = calloc(128, sizeof(image));
        for(i = 32; i < 127; ++i){
            char buff[256];
            sprintf(buff, "data/labels/%d_%d.png", i, j);
            alphabets[j][i] = load_image_color(buff, 0, 0);
        }
    }
    return alphabets;
}
```

After a while, I found there maybe a error in line 181:
`image **alphabets = calloc(nsize, sizeof(image));`

You should not alloc a nsize `image` array, instead it must be a nsize `image*` array.

**Comment by Grabber** [comment]
There are tons of errors like it.

**Comment by Kenneth-X** [comment]
yeah, my friends and i also find this bug and i am very curious how this code could work if these kinds of bug exist.

**Comment by Kenneth-X** [comment]
yeah, my friends and i also find this bug 
and i am very curious how this code could work and how the pictures are loaded if these kinds of bug exist.

**Comment by kangmingxuan** [comment]
@Kenneth-X This code applied more memory than needed, so it just happen to work OK.

Aha, master pieces sometimes work fine by coincidence. :-P
